DemoApp/src/Console.cc

```diff
@@ -1,8 +1,10 @@
 #include <sstream>
 #ifdef _WIN32
 #include <windows.h>
-#elif defined __unix__
+#elif defined(__unix__) || defined(unix) || defined(__unix) || defined(__linux__) || \
+    (defined(__APPLE__) && defined(__MACH__))
```

Comment:
To samo, nie wystarczy samo __APPLE__?

Reply:
Nie wiem, znalazłam to w mądrościach internetu :v